api/api.proto

```diff
@@ -2684,3 +2698,53 @@ message LocateResponse {
     // <containerid>: /var/www
     map<string, string> dockerids = 2;
 }
+
+message VolumePlacementSpec {
+  // Defines a list of rules as part of the placement spec
+  repeated VolumePlacementRule rules = 1;
+}
+
+message VolumePlacementRule {
+  // Defines the number of volume replicas affected by this rule. If not provided,
+  // rule would affect all replicas
+  // (optional)
+  int64 affected_replicas = 1;
```

Comment:
do we really need int64 for all the replicas? It should probably be int32, right?

Reply:
Changed to int32.